packages/account-tree-controller/src/backup-and-sync/syncing/group.ts

```diff
@@ -58,12 +58,18 @@ export const createMultichainAccountGroup = async (
       });
     }
   } catch (error) {
+    // This can happen if the Snap Keyring is not ready yet when invoking
```

Comment:
I suppose that most of the time this will be true, but what would happen if both EVM and Snap accounts are not created? Do we still recover later?